kunta-api-server/src/test/java/fi/otavanopisto/kuntaapi/test/server/integrations/mikkelinyt/MikkeliNytRestTestsIT.java

```diff
@@ -0,0 +1,348 @@
+    String eventImagePath = String.format("%s/1000/%s", imagesBasePath, attachmentId);
+    String eventThumb = String.format("%s%s", baseUrl, eventThumbPath);
+    String eventImage = String.format("%s%s", baseUrl, eventImagePath);
+    OffsetDateTime start = OffsetDateTime.of(2020, 5, 6, 17, 30, 0, 0, ZoneOffset.ofHours(3));
+    OffsetDateTime end = OffsetDateTime.of(2020, 5, 6, 19, 00, 0, 0, ZoneOffset.ofHours(3));
+    createIdentifier(kuntaApiOrganizationId, MikkeliNytConsts.IDENTIFIER_NAME, "test-source-id", IdType.ORGANIZATION.name());
+    createIdentifier(kuntaApiEventId, MikkeliNytConsts.IDENTIFIER_NAME, eventId, IdType.EVENT.name());
+    createIdentifier(kuntaApiAttachmentId, MikkeliNytConsts.IDENTIFIER_NAME, attachmentId, IdType.ATTACHMENT.name());
+    mocker.mockGetBinary("/" + eventThumbPath, "image/jpeg", "test-image-480.jpg");
+    mocker.mockGetBinary("/" + eventImagePath, "image/jpeg", "test-image-1000.jpg");
+
+    return mocker.mockEvent(kuntaApiOrganizationId, kuntaApiEventId, kuntaApiAttachmentId, eventId, eventUrl, eventName, eventDescription, eventCity, eventAddres, eventZip, eventPlace, eventImage, eventThumb, start, end);
+  }
+
+  private void cleanMock(MikkeliNytMocker mocker, String kuntaApiOrganizationId, String kuntaApiEventId, String kuntaApiAttachmentId) {
+    mocker.endMock();
+
+    deleteIndentifier(kuntaApiOrganizationId);
+    deleteIndentifier(kuntaApiEventId);
+    deleteIndentifier(kuntaApiAttachmentId);
+  }
+
+  private void createSettings(String kuntaApiOrganizationId, String baseUrl, String imagesBasePath) {
+    insertSystemSetting(MikkeliNytConsts.SYSTEM_SETTING_APIKEY, ORGANIZATION_SETTING_APIKEY);
+    insertOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_LOCATION, ORGANIZATION_SETTING_LOCATION);
+    insertOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_BASEURL, baseUrl);
+    insertOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_IMAGEBASEURL, String.format("%s/%s/1000/", baseUrl, imagesBasePath));
+  }
+  
+  private void deleteSettings(String kuntaApiOrganizationId) {
+    deleteSystemSetting(MikkeliNytConsts.SYSTEM_SETTING_APIKEY);
+    deleteOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_LOCATION);
+    deleteOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_BASEURL);
+    deleteOrganizationSetting(kuntaApiOrganizationId, MikkeliNytConsts.ORGANIZATION_SETTING_IMAGEBASEURL);
+  }
+  
+  public class MikkeliNytMocker extends AbstractMocker {
+    
+    public MikkeliNytMocker() {
+      this.events = new ArrayList<>();
+    }
+    
+    public Event mockEvent(String kuntaApiOrganizationId, String kuntaApiEventId, String kuntaApiAttachmentId, String id, String url, String name, String description, String city, String address, String zip, String place, String image, String thumb, OffsetDateTime start, OffsetDateTime end) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 15 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00107)